packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -171,6 +276,16 @@ const useBigNumberConfig = (
         selectedField,
         setSelectedField,
         getField,
+        comparisonValue,
+        showLabel,
+        setShowLabel,
+        showComparison,
+        setShowComparison,
+        comparisonFormatTypes: ComparisonFormatTypes,
```

Comment:
These enums can be imported into any file as they are right now. We can remove them from here.